discv5/discv5-wire.md

```diff
@@ -19,6 +18,9 @@ Here we present the notation that is used throughout this document.
     means binary XOR of `a` and `b`\
 `sha256(x)`\
     is the SHA256 digest of `x`\
```

Comment:
`rlp_bytes(x)` (above) doesn't look to be used anymore. M.b. remove it?

Reply:
Thanks, I will remove!